examples/frontend-js-with-bundle/index.js

```diff
@@ -15,8 +15,10 @@ const lblConnTo = document.getElementById('lblConnTo');
 const table = document.getElementById('fileTable');
 const alertDiv = document.getElementById('alertDiv');
 
-// import { Transport } from './cp210x-webusb.js'
-import * as esptooljs from "./bundle.js";
+// This is a frontend example of Esptool-JS using local bundle file
+// To optimize use a CDN hosted version like
+// https://unpkg.com/esptool-js@0.2.0/bundle.js
+import * as esptooljs from "../../bundle.js";
```

Comment:
why not just import from the CDN here ?

Reply:
oh I guess it's an example, we want it to be from there. The problem with relative URL is that now you can't have this folder be exported to github pages, as `..` doesn't exist anymore.

You could always consider doing a hybrid approach [like this](https://github.com/esphome/esp-web-tools/blob/main/index.html#L139):

```html
    <script module>
      import(
        // In development we import locally.
        window.location.hostname === "localhost"
          ? "/dist/web/install-button.js"
          : "https://unpkg.com/esp-web-tools/dist/web/install-button.js?module"
      );
    </script>
```